src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -51,6 +51,40 @@ class PumpConfigurationAndControlAttrAccess : public AttributeAccessInterface
 
 PumpConfigurationAndControlAttrAccess gAttrAccess;
 
+CHIP_ERROR PumpConfigurationAndControlAttrAccess::Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder)
+{
+    emberAfDebugPrintln("Reading from PCC");
+
+    VerifyOrDie(aPath.mClusterId == PumpConfigurationAndControl::Id);
```

Comment:
How would Die here ?

Reply:
Good question, looks like this is also used in some other clusters. In general I'm not sure if these Read & Write functions actually is getting used 🤔

Reply:
The PumpConfigurationAndControlAttrAccess object does contain the cluster id (PumpConfigurationAndControl::Id), so I think the idea here is that if PumpConfigurationAndControlAttrAccess::Read or Write is called and the aPath.mClusterId is not for this cluster, something has messed up?

Reply:
In general, it does not look like this cluster is utilizing PumpConfigurationAndControlAttrAccess that much, so I assume the plan was to provide an empty handler which can be used later if needed.

But if we want to clean up, it should not be a big deal to remove it, since it is not doing anything.

Reply:
Looking at the original PR, this was added with the intention of being used later on, but since we are still not actively using it I have removed it again.